### PR TITLE
GC Features: JS equalheight to CSS equalheight

### PR DESCRIPTION
**Pull request by Garneauma**
Changing GC Features component to use CSS equalheight feature instead of its JavaScript equivalent.

Related to WET-269

### CODE REVIEW

File: components/gc-features/_base.scss
Comment (duboisp) [inline components/gc-features/_base.scss:25]: This need to be tested: Text-zoom. But we should not need to use any "height:100%".

More rational will be needed to understand if this don't reveal a bug with the equal height CSS feature we have.

File: components/gc-features/gc-features-en.html
Comment (duboisp) [inline components/gc-features/gc-features-en.html:44]: The old/deprecated pattern need to be documented (working example)

**Review by duboisp — CHANGES_REQUESTED** [review]
The old/deprecated pattern need to be documented (working example)

File: components/gc-features/_base.scss
Comment (Garneauma) [reply components/gc-features/_base.scss:25]: Removed `height: 100%` and reinstated `.eqht-trgt`

**Review by duboisp — CHANGES_REQUESTED** [review]
The component version need to be majored to v5.0.

Mention that Version 4.0 are still valid and supported.

**Comment by duboisp** [comment]
Approved once local testing completed and the documentation change about the component version is completed.

File: components/gc-features/gc-features-en.html
Comment (duboisp) [inline components/gc-features/gc-features-en.html:36]:
```suggestion
	<dd>Version 5.0</dd>
	<dt>Supported working version</dt>
	<dd>
		<ul>
			<li>Version 5</li>
			<li>Version 4</li>
			<li>Version 3 (deprecated)</li>
			<li>Version 2 (deprecated)</li>
			<li>Version 1 (deprecated)</li>
		</ul>
	</dd>
```

**Review by duboisp — CHANGES_REQUESTED** [review]
Just one small thing, let's specify we support version 4 and version 5. And that older version are deprecated. That way we will be able to mark this change as a minor change.

I tested locally and work as expected. Once the little content change we will merge it.

File: components/gc-features/gc-features-fr.html
Comment (duboisp) [inline components/gc-features/gc-features-fr.html:38]:
```suggestion
	<dt>Version fonctionnelle compatible</dt>
```

**Review by duboisp — CHANGES_REQUESTED** [review]

File: components/gc-features/gc-features-fr.html
Comment (duboisp) [inline components/gc-features/gc-features-fr.html:45]:
```suggestion
	<dt>Version  compatible et fonctionnelle</dt>
	<dd>
		<ul>
			<li>Version 5</li>
			<li>Version 4</li>
			<li>Version 3 (désuet)</li>
			<li>Version 2 (désuet)</li>
			<li>Version 1 (désuet)</li>
```

**Review by duboisp — CHANGES_REQUESTED** [review]

**Review by duboisp — APPROVED** [review]

**Comment by duboisp** [comment]
This is a minor change

**Pull request merged**
